modules/gapi/src/backends/cpu/gcpuimgproc.cpp

```diff
@@ -244,6 +244,15 @@ GAPI_OCV_KERNEL(GCPURGB2GrayCustom, cv::gapi::imgproc::GRGB2GrayCustom)
     }
 };
 
+GAPI_OCV_KERNEL(GCPUnormalize, cv::gapi::imgproc::Gnormalize)
```

Comment:
GCPUnormalize -> GCPUNormalize, Gnormalize -> GNormalize. (cv::gapi::normalize(..) should remain as is)